front/lib/models/assistant/conversation.ts

```diff
@@ -486,7 +508,7 @@ Message.belongsTo(UserMessage, {
 });
 
 AgentMessage.hasOne(Message, {
-  as: "agentMessage",
+  as: "message",
```

Comment:
why changing that ? agree the name could be message but it's explicitly set like this and we have the UserMessage->Message relation in the same format